README.md

```diff
@@ -47,10 +43,11 @@ More details about lambda deployments are available in the [AWS documentation](h
 Simply use our decorator to report metrics:
 
 ```python
-from iopipe.iopipe import IOpipe
+from iopipe import IOpipe
+
 iopipe = IOpipe("your-client-id")
```

Comment:
Can we use this opportunity to update this to our preferred "token" language? (see node agent if you want a reference) would be neat to carry through in the code as well.

Reply:
Scratch that, opened a separate issue #103